package.json

```diff
@@ -32,6 +32,7 @@
     "tape": "^4.5.1"
   },
   "dependencies": {
+    "@google/cloud-diagnostics-common": "0.2.3",
     "google-auth-library": "0.9.8",
     "lodash": "4.13.1"
```

Comment:
Any reason for pinning the version for lodash and google-auth-library? I agree with pinning diagnostics-common.